examples/custom_notification.rs

```diff
@@ -45,6 +45,8 @@ impl LanguageServer for Backend {
                 }),
                 ..ServerCapabilities::default()
             },
+						#[cfg(feature = "proposed")]
+            offset_encoding: None,
```

Comment:
I'm fairly sure this type implements `Default`. Any possibility we could replace these `#[cfg]` annotated fields with a `..Default::default()` initializer at the bottom instead?